client/my-sites/checkout/checkout-thank-you/pending/index.tsx

```diff
@@ -37,6 +37,11 @@ interface CheckoutPendingProps {
 	receiptId: number | undefined;
 	siteSlug?: string;
 	redirectTo?: string;
+	// `fromSiteSlug` is the Jetpack site slug passed from the site via url query arg (into
+	// checkout), for use cases when there is not a site in context, such as siteless checkout.
+	// As opposed to `siteSlug` which is the site slug present when the site is in context
+	// (ie- the site is available in state, such as when site is connected and user logged in).
+	fromSiteSlug?: string;
```

Comment:
This is relatively minor, but at least in my IDE you have to write property comments using multi-line comment style in order to get them to show up when examining the property.

Right now I see this:
<img width="567" alt="Screenshot 2023-09-27 at 11 04 00 AM" src="https://github.com/Automattic/wp-calypso/assets/2036909/716c8de7-f275-47fe-8ecd-9d34f3c370b6">

But if I apply the following change, I see this instead:
<img width="653" alt="Screenshot 2023-09-27 at 11 02 18 AM" src="https://github.com/Automattic/wp-calypso/assets/2036909/736867d3-cd94-4b12-a0b4-921ed224e201">




```suggestion
	/**
	 * `fromSiteSlug` is the Jetpack site slug passed from the site via url
	 * query arg (into checkout), for use cases when there is not a site in
	 * context, such as siteless checkout. As opposed to `siteSlug` which is
	 * the site slug present when the site is in context (ie- the site is
	 * available in state, such as when site is connected and user logged in).
	 */
	fromSiteSlug?: string;
```

Reply:
Ahh yes, I see. **TIL!**  Ok yes, will do, thanks @sirbrillig!